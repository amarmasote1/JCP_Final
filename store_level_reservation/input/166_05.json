[
  {
    "storeID": "2015",
    "itemID": "15345672202",
    "uom": "EACH",
    "supplyData": {
      "onhandsupply": 0
    },
    "demandData": {
        "reserveddemand": 0
      },
    "timestamp": "<$currentTime>"
    
  },
  {
    "storeID": "2018",
    "itemID": "15345672202",
    "uom": "EACH",
    "supplyData": {
      "onhandsupply": 0
    },
    "demandData": {
        "reserveddemand": 0
      },
    "timestamp": "<$currentTime>"
    
  },
  {
    "storeID": "2020",
    "itemID": "15345672202",
    "uom": "EACH",
    "supplyData": {
      "onhandsupply": 0
    },
    "demandData": {
        "reserveddemand": 0
      },
    "timestamp": "<$currentTime>"
    
  },
  {
    "storeID": "2015",
    "itemID": "15345672203",
    "uom": "EACH",
    "supplyData": {
      "onhandsupply": 0
    },
    "demandData": {
        "reserveddemand": 0
      },
    "timestamp": "<$currentTime>"
    
  },
  {
    "storeID": "2018",
    "itemID": "15345672203",
    "uom": "EACH",
    "supplyData": {
      "onhandsupply": 0
    },
    "demandData": {
        "reserveddemand": 0
      },
    "timestamp": "<$currentTime>"
    
  },
  {
    "storeID": "2020",
    "itemID": "15345672203",
    "uom": "EACH",
    "supplyData": {
      "onhandsupply": 0
    },
    "demandData": {
        "reserveddemand": 0
      },
    "timestamp": "<$currentTime>"
    
  }
]