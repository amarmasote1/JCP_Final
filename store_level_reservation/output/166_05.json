{
    "message": "Success",
    "body": [
        {
            "itemID": "15345672202",
            "uom": "EACH",
            "demandData": {
                "reserveddemand": 0
            },
            "markDown": {
                "active": false
            },
            "storeID": "2015",
            "supplyData": {
                "onhandsupply": 0
            },
            "timestamp": "2020-03-03 11:53:34.0000 IST"
        },
        {
            "itemID": "15345672202",
            "uom": "EACH",
            "demandData": {
                "reserveddemand": 0
            },
            "markDown": {
                "active": false
            },
            "storeID": "2018",
            "supplyData": {
                "onhandsupply": 0
            },
            "timestamp": "2020-03-03 11:53:34.0000 IST"
        },
        {
            "itemID": "15345672202",
            "uom": "EACH",
            "demandData": {
                "reserveddemand": 0
            },
            "markDown": {
                "active": false
            },
            "storeID": "2020",
            "supplyData": {
                "onhandsupply": 0
            },
            "timestamp": "2020-03-03 11:53:34.0000 IST"
        },
        {
            "itemID": "15345672203",
            "uom": "EACH",
            "demandData": {
                "reserveddemand": 0
            },
            "markDown": {
                "active": false
            },
            "storeID": "2015",
            "supplyData": {
                "onhandsupply": 0
            },
            "timestamp": "2020-03-03 11:53:34.0000 IST"
        },
        {
            "itemID": "15345672203",
            "uom": "EACH",
            "demandData": {
                "reserveddemand": 0
            },
            "markDown": {
                "active": false
            },
            "storeID": "2018",
            "supplyData": {
                "onhandsupply": 0
            },
            "timestamp": "2020-03-03 11:53:34.0000 IST"
        },
        {
            "itemID": "15345672203",
            "uom": "EACH",
            "demandData": {
                "reserveddemand": 0
            },
            "markDown": {
                "active": false
            },
            "storeID": "2020",
            "supplyData": {
                "onhandsupply": 0
            },
            "timestamp": "2020-03-03 11:53:34.0000 IST"
        }
    ],
    "status": 200
}