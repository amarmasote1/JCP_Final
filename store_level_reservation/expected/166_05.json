{"status":200,"message":"Success","body":[{"itemID":"15345672202","uom":"EACH","storeID":"2015","supplyData":{"onhandsupply":0},"demandData":{"reserveddemand":0}},{"itemID":"15345672202","uom":"EACH","storeID":"2018","supplyData":{"onhandsupply":0},"demandData":{"reserveddemand":0}},{"itemID":"15345672202","uom":"EACH","storeID":"2020","supplyData":{"onhandsupply":0},"demandData":{"reserveddemand":0}},{"itemID":"15345672203","uom":"EACH","storeID":"2015","supplyData":{"onhandsupply":0},"demandData":{"reserveddemand":0}},{"itemID":"15345672203","uom":"EACH","storeID":"2018","supplyData":{"onhandsupply":0},"demandData":{"reserveddemand":0}},{"itemID":"15345672203","uom":"EACH","storeID":"2020","supplyData":{"onhandsupply":0},"demandData":{"reserveddemand":0}}]}